maestro-client/src/main/java/maestro/js/GraalJsEngine.kt

```diff
@@ -70,6 +73,17 @@ class GraalJsEngine(
         return createContext().eval(source)
     }
 
+    private fun Context.Builder.setAccessConfigFromEnv(): Context.Builder {
+        val allowAllAccess = Env.getSystemEnv(ALL_ACCESS_ENV).evalTruthness()
+        val allowHostAccess = Env.getSystemEnv(HOST_ACCESS_ENV).evalTruthness()
+        val allowClassLookup = Env.getSystemEnv(CLASS_LOOKUP_ENV).evalTruthness()
```

Comment:
Instead of introducing a concept of "truthness", let's consider the option to be enabled if the env var is set to any value – even if that value is an empty string.